.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+<!-- Please complete the following sections as necessary. -->
+
+## Description
+
+<!-- Summary of the changes, related issue, relevant motivation, and context -->
+
+### Ticket
+
+<!-- Link to ticket in pivotal -->
```

Comment:
I think this is unncecessary - it should be in every commit message. no need to ask people to copy-paste it again

Reply:
I hear that. I see this as a small convenience for the person I'm asking to review my work, but I'm not trying to die on this hill either.

Reply:
I have seen this used to link to multiple tickets, say for instance a fix addresses multiple bugs.

Reply:
It happens infrequently, but I think that when a commit does address multiple tickets, it should link to all of them in the commit message. It could ALSO be here, sure, but it should also be in the commit message I think

Reply:
I think its kind of nice if you're reading this and then you can just click the link as a reviewer (Although its slightly more work for the person writing the PR). But also I'm don't super care which we go with here. 

Also when we've done this in the past we have the link without the story ID, so the PR review writer just needs to add the end of the URL.

Reply:
Keep it optional?

Reply:
I think having a link directly to pivotal would slightly make it more for the reviewer so I'm for it but I also don't feel too strongly about it.